core/frontend/src/tile/map/MapTile.ts

```diff
@@ -131,6 +131,7 @@ const scratchCorners = [Point3d.createZero(), Point3d.createZero(), Point3d.crea
 export class MapTile extends RealityTile {
   private static _maxParentHeightDepth = 4;
   private _imageryTiles?: ImageryMapTile[];
+  private _hiddenTiles?: ImageryMapTile[];
```

Comment:
Does this record *every* hidden tile? Is that necessary, or do you only need to know that at least one tile is hidden?

Reply:
Yes it records every hidden tile, and used by MapTileTree.reportTileVisibility.  The imagery tile tree range visibility state is independent from the actual map-layer visibility state; which means you can explicitly hide a layer, but you will still get events if the layer goes out of range by zoom-in/out.  So we need to inspect **every** selected tile, no matters if it's hidden or not.